tests/test_download.py

```diff
@@ -1,19 +1,26 @@
-import uuid
-import pytest
 import json
-import pandas as pd
-
+import uuid
+from pathlib import Path
 from tempfile import TemporaryDirectory
 
-from gtfs_rt_validator_api import download_gtfs_schedule_zip, download_rt_files, validate, validate_gcs_bucket, validate_gcs_bucket_many
+import pandas as pd
+import pytest
 from calitp.storage import get_fs
-from pathlib import Path
+
+from gtfs_rt_validator_api import (
+    download_gtfs_schedule_zip,
+    download_rt_files,
+    validate,
+    validate_gcs_bucket,
+    validate_gcs_bucket_many,
+)
 
 # Note that data for these tests is seeded by scripts/seed_bucket_data.py
-GCS_BASE_DIR="gs://calitp-py-ci/gtfs-rt-validator-api"
+GCS_BASE_DIR = "gs://calitp-py-ci/gtfs-rt-validator-api"
```

Comment:
Just noticed this thing that still points to the `ci` directory.... Should we comment that this file is deprecated too?

Reply:
yeah.... honestly the tests shouldn't hit GCS at all

Reply:
I added a note that the tests need to be reworked. We probably need some pytest fixtures to help with this stuff.